### DISCUSSION TITLE
Cursor freezes every few seconds.

### DISCUSSION THREAD

**Discussion by Cwumpet** [ppy/osu#19588] [Q&A] [answered]
I'm using a Wacom tablet with Wacom default drivers and it works perfectly fine on stable. I have the tablet option disabled, and use full area with 2.55 sensitivity. My mouse works fine but my tablet cursor freezes every few seconds. It hasn't stopped.

https://user-images.githubusercontent.com/75339736/183063580-bd27d49d-5046-470f-834a-aba15924cad4.mp4

I can still press the pen nib and it makes an input even when the cursor is frozen. This is very weird. Any help would be SUUUPER appreciated!

Edit: I forgot to say it works fine with high precision mouse turned off but I use full tablet area so I cant play without sensitivity.

**Comment by nagi-desuuu** [comment]
Are you sure you have your driver's updated to their latest version?
The way you said default drivers makes me think that you haven't updated them in a while.

**Reply by Cwumpet** [reply]
nope, my drivers are up to date and work in every other instance.

**Comment by peppy** [comment]
This could be anything really. But from the frame graphs it definitely looks like not something lazer's end.

Have you tried the built-in lazer tablet support? Does that work without the freezes?

**Reply by Cwumpet** [reply]
I couldn't find a way to convert my full area + sensitivity to a tablet area that felt normal and also i cant input decimal values (I think?)

**Comment by Boo-BooLazer** [comment]
Hi, I posted a issue pretty much identical to yours, and am having the exact same problem. First off, did you have any luck fixing it? Second, what tablet are you using? I am using the CTH-460.

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/19960
